### DISCUSSION TITLE
upgrade openfast version

### DISCUSSION THREAD

**Discussion by letuyen2101** [OpenFAST/openfast#1197] [Q&A] [answered]
Hello everyone,
I installed openfast in ubuntu through anaconda successfully, then the command: openfast -v >> give OpenFAST-v2.4.0. I then download  new version: openfast-3.2.0.zip and install. After that, the command: openfast -v >> still give OpenFAST-v2.4.0. How i delete old version and upgrade the new one. ?
Appreciated !

**Comment by rafmudaf** [comment, marked as answer]
Hi @letuyen2101 youve got to upgrade through conda: `conda update openfast`.
The reason you aren't seeing the correct version is because the OpenFAST zip file that you downloaded contains the source code and not the binary. If it did have the binary, it wouldn't be in your path. The conda installation method handles all of these things better.

**Reply by letuyen2101** [reply]
Thanks rafmudaf for your prompt reply.
my path ../openFast/install/bin:$PATH", the openfast -v command gives me:
OpenFAST-HEAD-HASH-NOTFOUND

Is it correct ?
Thanks

**Reply by rafmudaf** [reply]
Yes it's ok. It's a nuance of the conda build system, and we should fix this in the future.
